recipes/imagl/all/conanfile.py

```diff
@@ -14,8 +14,8 @@ class ImaglConan(ConanFile):
     description = "A lightweight library to load image for OpenGL application."
     topics = ("opengl", "texture", "image")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False], "with_png": [True, False]}
-    default_options = {"shared": False, "fPIC": True, "with_png": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "with_png": [True, False], "with_jpeg": [True, False], "not_in_cci": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_png": True, "with_jpeg": True, "not_in_cci": False}
```

Comment:
We are trying to stick with "positive options" would the logic be flipped?

Reply:
Perhaps call it "relaxed_compiler_check" ? this way it's less questionable?

Reply:
I prefer to call it "allow_clang_11" since it's only for this case. Would it be OK?